avm/res/insights/action-group/tests/e2e/defaults/main.test.bicep

```
@@ -0,0 +1,43 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-insights.actiongroups-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'iagdef'
```

```suggestion
param serviceShort string = 'iagmin'
```

To work with the PSRule config. Requires a refresher of the readme.